ci/lint_stack.sh

```diff
@@ -22,7 +22,8 @@ print_list() {
 
 # Get existing stack names from local files
 get_local_stack_names() {
-  stack_names=( $(/bin/grep -r -w -h '^stack_name:' ${PATH} | /usr/bin/cut -d':' -f2 | /usr/bin/awk '{$1=$1};1') )
+  # get stack names and strip spaces and tabs from string
+  stack_names=( $(/bin/grep -r -w -h '^stack_name:' ${PATH} | /usr/bin/cut -d':' -f2 | /usr/bin/tr -d '\040\011\042\047') )
```

Comment:
Looks good -- only comment is, I have to go look up what these octals are!

Reply:
just so i remember myself.  https://web.cs.dal.ca/~zyu/ascii.html